src/collective/cover/testing.py

```diff
@@ -104,9 +104,9 @@ def generate_jpeg(width, height):
                 if abs(z) > 2.0:
                     break
                 z = z * z + c
-            r = i % 4 * 64
-            g = i % 8 * 32
-            b = i % 16 * 16
+            r = i % 4 * 64  # noqa: S001
```

Comment:
don't include this as this bug is already fixed: https://github.com/gforcada/flake8-pep3101/issues/16

what version of `flake8-pep3101` are you using?

Reply:
I don't know how to check this out